### ISSUE TITLE
PyPI invite

### ISSUE THREAD

**Issue by SimonBiggs** [darcymason/egsnrc2py#10]
Hi @darcymason,

I think it would make sense that  https://github.com/darcymason/egsnrc2py/tree/main/egsnrc2py/egsnrc becomes the source for https://pypi.org/project/egsnrc/. I've pulled you into that PyPI repo to that end.

I've been a little quite on the EGSnrc -> python front for the last week or so, I suspect that will continue for a little longer before I can jump back in.

Cheers :),
Simon

**Comment by darcymason** [comment]
Hi @SimonBiggs,

Sorry if I stepped on an existing name.  There are limited ways of putting "py" and "egsnrc" together and in the end I felt that if (big if) the Python version ever became the reference implementation, the simple name was best as a working name.

Thanks for the PyPI invites.

> I've been a little quite on the EGSnrc -> python front for the last week or so,

No worries at all - I've been slowing down too, but hopeful of maybe getting something to test run before too long.  BTW in the plan in my head, it won't start with any compatibility for GPU - I'd like to get something (ELECTR) working and correct (identical results to Mortran EGSnrc if possible)  with the fewest complications initially - it's enough to make sure nothing was dropped and all the statements are at the right indent level.  I may even switch back to mostly global variables at first.

**Comment by SimonBiggs** [comment]
> BTW in the plan in my head, it won't start with any compatibility for GPU

Yup, that's in my head too. In fact, it might be that once you have the Python version of something running, that's what I can build on/around to bring in GPU capability.

> Sorry if I stepped on an existing name.

All good, it's perfect. I figure `egsnrc` should be the name on PyPI and imports.
